code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -10,7 +10,7 @@
  */
 /obj/item/clothing/suit/bluetag
 	name = "blue laser tag armour"
-	desc = "Blue Pride, Galaxy Wide."
+	desc = "Blue Pride, World Wide."
```

Comment:
Poorly worded, also why the fuck do we need a generic blue tag?

Reply:
because... 'm fixing it.